src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1598,67 +1691,66 @@ subroutine apply_calving_velocity(meshPool, geometryPool, velocityPool, domain,
                    .not. li_mask_is_ice(cellMask(jCell)) .and. bedTopography(jCell) < config_sea_level & ! ensure margin is w/ocn
                    ) then
                   edgeLengthScaling = scale_edge_length(angleEdge(iEdge), uReconstructX(1,iCell), uReconstructY(1,iCell))
-                  requiredCalvingVolumeNonDynEdge(iEdge) = calvingVelocity(iCell) * &
-                          edgeLengthScaling * dvEdge(iEdge) * thicknessForCalving(iCell) * deltat
-                  requiredCalvingVolumeNonDynCell(iCell) = requiredCalvingVolumeNonDynCell(iCell) + &
-                          requiredCalvingVolumeNonDynEdge(iEdge) ! Keep running total
+                  requiredAblationVolumeNonDynEdge(iEdge) = ablationVelocity(iCell) * &
+                          edgeLengthScaling * dvEdge(iEdge) * thicknessForAblation(iCell) * deltat
+                  requiredAblationVolumeNonDynCell(iCell) = requiredAblationVolumeNonDynCell(iCell) + &
+                          requiredAblationVolumeNonDynEdge(iEdge) ! Keep running total
                endif
             enddo
-            ! c. Calculate calvingThickness here
-            removeVolumeHere = min(cellVolume(iCell), requiredCalvingVolumeNonDynCell(iCell))  ! Don't use more than available
-            calvingThickness(iCell) = removeVolumeHere / areaCell(iCell)
-            calvedVolumeNonDynCell(iCell) = removeVolumeHere
-            uncalvedVolumeNonDynCell(iCell) = requiredCalvingVolumeNonDynCell(iCell) - removeVolumeHere
+            ! c. Apply calving here
```

Comment:
Return this comment to "c. Calculate ablationThickness here".  The distinction being we don't actually apply calving until the end of this routine when thickness is modified.

Reply:
Done